src/Module/User/Api/IdentityTokenService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Module\User\Api;
+
+use App\Module\User\Domain\Entity\Token;
+use Yiisoft\Auth\IdentityInterface;
+
+interface IdentityTokenService
+{
+    public function generateIdentityToken(IdentityInterface $identity, string $type): Token;
+    public function addIdentityToken(IdentityInterface $identity, string $token, string $type): Token;
```

Comment:
```suggestion
    public function generate(IdentityInterface $identity, string $type): Token;
    public function add(IdentityInterface $identity, string $token, string $type): Token;
```

Сервис уже имеет префикс, зачем его дублировать в методах?

Reply:
Несколько интерфейсов могут ссылаться на один и тот же сервис в реализации. Лучше с перфиксами